simple-async/README.md

```diff
@@ -0,0 +1,183 @@
+## Simple Async template
+
+The simple template will create the following project structure:
+
+```text
+src/
+├── app.rs     -> holds the state and application logic
+├── event.rs   -> handles the terminal events (key press, mouse click, resize, etc.)
+├── handler.rs -> handles the key press events and updates the application
+├── lib.rs     -> module definitions
+├── main.rs    -> entry-point
+├── tui.rs     -> initializes/exits the terminal interface
+└── ui.rs      -> renders the widgets / UI
+```
+
+This is identical to the simple template but has `async` events out of the box with `tokio` and
+`crossterm`'s `EventStream`.
+
+Here's the exact diff if you want to convert your own code to async:
```

Comment:
Could we give the command or a link to this diff? This is the kind of thing that will not get updated.

Reply:
I did `diff -bur simple simple-async`. 

I don't think this diff ever needs to be updated, it's just a reference for code that works when converting from `sync` to `async`.

Reply:
Alright if you think it won't change.